perceval_interop/utils/exceptions.py

```diff
@@ -0,0 +1,44 @@
+# MIT License
+#
+# Copyright (c) 2025 Quandela
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+class ConversionBadVersionError(Exception):
+    pass
+
+
+class ConversionSyntaxError(Exception):
+    pass
+
+
+class ConversionUnsupportedFeatureError(Exception):
+    pass
+
+
+class MissingDependency(Exception):
+
+    def __init__(self, self_name: str, extra_requirement: str):
+        super().__init__(f"{self_name} can't be imported: run 'pip install perceval_interop[{extra_requirement}]'")
+
+    def __call__(self, *args, **kwargs):  # Mimics the __init__ from the missing object
+        raise self
+
+    def __getattr__(self, item):
+        raise self
```

Comment:
We need to still allow getting regular attributes from Exception so packages that make use of Exceptions can still work

```
        if not item.startswith("__") and item not in {"add_note", "with_traceback"}:
            raise self
        return super().__getattribute__(item)
```

According to the pycharm autocompletion, there is also an "mro" attribute, but I couldn't manage to make it work

Reply:
Another way would be to keep in mind if the Exception has been raised, and if it does, use the regular __getattr__